test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments for long running tests proceed as following:
+
+1. Clone the following clusterloader2 repo. It's the tool used to deploy the benchmark workload in a KubeStellar environment:
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+   The script `test/performance/common/setup-clusterloader2.sh` uses the environment variable set above. 
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script to set-up your environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd $HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash 
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory.
+
+   ```bash 
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic.
+   
+   ```bash 
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numberOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
+   - burstSize: number of workload objects created at a burst event (*default value: "1"*) 
+   - stepDelay: specifies the time interval between burst events (*default value: "60"*)
+
+   The workload objects are created using the clusterloader `SteppedLoad` tuningSet. It defines a load that generates a burst of a given size every X seconds, see more [here](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-se). To learn more about clusterloader2 tuningSet, see the following: [tuningSets configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
```

Comment:
(A) The first URL here is missing the "t" at the end of the fragment identifier.

(B) The second URL here points to the definition of `PoissonLoad`, but don't you want to point to `SteppedLoad` instead?

(C) These URLs seem swapped. Shouldn't the first point to the definition of SteppedLoad, and the second point to the documentation of the concept of tuning set?

Reply:
Good catch! You're correct, the URL were swapped indeed. Just fixed it!